recipe/meta.yaml

```diff
@@ -13,21 +13,24 @@ source:
 build:
   number: 0
   noarch: python
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
   entry_points:
     - conda-pack = conda_pack.cli:main
 
 requirements:
   host:
     - python
     - pip
+    - wheel
   run:
-    - setuptools
     - python
```

Comment:
```suggestion
    - setuptools
    - python
```
There is one mention of setuptools in the code. Since we don't run the test, better keep it.
https://github.com/conda/conda-pack/blob/0.7.1/conda_pack/core.py#L1179
https://github.com/conda/conda-pack/blob/0.7.1/setup.py#L32

Reply:
Good catch.